app/models/grda_warehouse/hud/enrollment.rb

```diff
@@ -150,6 +150,9 @@ class Enrollment < Base
     end
 
     scope :visible_in_window_to, ->(user) do
+      # FIXME: is this correct? visible_to also includes logic to include
+      # visible_in_window, but is this supposed to only return window data?
+      # like this, I believe it returns both window and assigned data
```

Comment:
Does this need to be addressed? Or, since it is already existing behavior, can it be deferred?

Reply:
I'll just note that it includes data a user can see, I _think_ that's the intention now, whereas before it was to grant access to a specific set of data.